annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodRecord.java

```diff
@@ -40,8 +41,8 @@ public class MethodRecord {
   public Integer id;
   /** Location of the containing method. */
   public OnMethod location;
-  /** Is true if the method is already annotated with {@code Nullable} annotation. */
-  public boolean hasNullableAnnotation;
+  /** Set of annotations on method return type */
```

Comment:
Will this include both declaration annotations on the method and type use annotations on the return type?  Even if we don't support type use annotations right now it'd be good to know if we want all annotations to be in the same list or not.  Note that syntactically I think such annotations can be interleaved and appear in any order (and also interleaved with qualifiers like `public`)

Reply:
I think we need to support type use annotations at least for `@Nullable`, no? Otherwise we will have trouble when we move to JSpecify, so I'd say we want to make sure both cases are captured. Maybe worth a test with Jspecify's version of `@Nullable`?

Reply:
I looked over the relevant `SymbolUtil` method and I think this code will capture all annotations (both declaration and type use).  So I think we are ok for now.  When _injecting_ a type use annotation on a return type, we should try to put the annotation next to the return type if we can.  But that's separate from the issue being solved here.